dspace-api/src/test/java/org/dspace/content/CollectionTest.java

```diff
@@ -103,6 +103,7 @@ public void destroy()
                 if(collection != null)
                 {
                     collectionService.delete(context, collection);
+                    context.commit();
```

Comment:
This is an odd place for a `commit()` (between a collection deletion & a community deletion). Could we add a comment here as to why this is needed? Is it because the Collection needs to be fully deleted prior to the Community? Or is this a mistake?